perceval/backends/core/discourse.py

```diff
@@ -16,10 +16,12 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 # Authors:
-#    Santiago Dueñas <sduenas@bitergia.com>
-#    J. Manrique López de la Fuente <jsmanrique@bitergia.com>
```

Comment:
I checked the [commit history of discourse.py](https://github.com/chaoss/grimoirelab-perceval/commits/master/perceval/backends/core/discourse.py), but couldn't find any reference of jsmanrique as an author.

Reply:
I checked the history too and I don't see him. The change that introduced it is https://github.com/chaoss/grimoirelab-perceval/commit/d36293a4ba3cd4e6c95be900c145c09bf57c6f00.

I would leave the name there, probably he helped in writing the file, but it was committed by someone else.

Reply:
Thanks for the reply. I will revert the changes.

Reply:
thanks! (I hope this isn't too much work)

Reply:
> thanks! (I hope this isn't too much work)

Not really but, I am worried that the script won't work well in such corner cases.

Reply:
I see, I'd rely on the script and see how many corner cases pop up.